tests/decorators/test_decorators.py

```diff
@@ -201,6 +201,48 @@ def dummy():
                                          "connection": "dummy_conn"
                                      }]})
 
+    def test_queue_trigger(self):
+        app = self.func_app
+
+        @app.on_queue_change(arg_name="req", queue_name="dummy_queue",
+                             connection="dummy_conn")
+        def dummy():
+            pass
+
+        func = self._get_func(app)
+
+        trigger = func.get_bindings()[0]
```

Comment:
minor: add binding list length check, for all of tests

Reply:
updated as per recommendation. Thanks